ci/tests/backend/alba.py

```diff
@@ -240,8 +245,31 @@ def is_bucket_count_valid_with_policy(bucket_count, policies):
     return safe
 
 
+def initialise_disks(alba_node):
+    disks_to_init = [d['name'] for d in alba_node.all_disks if d['available'] is True]
```

Comment:
initialise should also take nr_of_disks and disk_type into account, similar to claim_disks